release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -778,13 +826,13 @@ In the following tables, features are marked with the following statuses:
 
```

Comment:
@lpettyjo Do we need to keep filestore CSI in the TP table since it's now GA for 3 consecutive releases?

Reply:
Yes. That's kinda how we do it. For 4.17, it will leave the list.

Reply:
ok all good then !